modal/functions.py

```diff
@@ -316,18 +317,27 @@ async def drain_input_generator():
 
     async def pump_inputs():
         nonlocal have_all_inputs
+        lats = []
+        sent_inputs = 0
         async for items in queue_batch_iterator(input_queue, MAP_INVOCATION_CHUNK_SIZE):
             request = api_pb2.FunctionPutInputsRequest(
                 function_id=function_id, inputs=items, function_call_id=function_call_id
             )
+            sent_inputs += len(items)
+            t0 = time.monotonic()
             resp = await retry_transient_errors(
                 client.stub.FunctionPutInputs,
                 request,
                 max_retries=None,
             )
+            lat = time.monotonic() - t0
+            lats.append(lat)
+            print("Sent", sent_inputs, lat)
```

Comment:
I'm getting this when running anything - intentional?

Reply:
Nope, unintentional and unrelated debug message